pymongo/monitor.py

```diff
@@ -29,24 +31,31 @@
 from pymongo.server_description import ServerDescription
 from pymongo.srv_resolver import _SrvResolver
 
+if TYPE_CHECKING:
+    from OpenSSL.SSL import Connection
```

Comment:
This should be a pymongo.pool.Connection.

Reply:
makes sense, fixed!